contrib/txsend/README.md

```diff
@@ -0,0 +1,26 @@
+## -txsend scripts
+
+These scripts can be used with the `-txsend` config option to send transactions
+via an external service instead of directly through the Zcash P2P network.
+
+### `zcash_to_i2pbote.py`
+
+Sends transactions as [I2P-Bote](https://i2pbote.xyz) messages to a remote
+listener, for them to broadcast to the Zcash network.
+
+#### Usage
+
+- Install [I2P](https://geti2p.net/en/download).
+- Install [I2P-Bote](https://i2pbote.xyz/install/).
```

Comment:
By instructions the plugin url is http://bote.i2p/i2pbote.su3 however the link seems to not be working.

I am new to i2p so i might be doing something wrong.

Reply:
By the way i do have the proxy configured in my browser but still it seems i cant connect to anything including the plugin url. Probably not something to discuss here.

Reply:
This URL is dead and will need to be updated.